src/pages/Admin/License/Create/index.jsx

```diff
@@ -0,0 +1,185 @@
+/*
+ Copyright (C) 2021 Shruti Agarwal (mail2shruti.ag@gmail.com)
+
+ SPDX-License-Identifier: GPL-2.0
+
+ This program is free software; you can redistribute it and/or
+ modify it under the terms of the GNU General Public License
+ version 2 as published by the Free Software Foundation.
+ This program is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ GNU General Public License for more details.
+
+ You should have received a copy of the GNU General Public License along
+ with this program; if not, write to the Free Software Foundation, Inc.,
+ 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+*/
+
+import React from "react";
+import InputContainer from "../../../../components/Widgets/Input";
+import Button from "../../../../components/Widgets/Button";
+
+const AddLicense = () => {
+  const handleChange = () => {};
+  const handleSubmit = (e) => {
+    e.preventDefault();
+  };
+  return (
+    <div className="main-container my-3">
+      <div className="row">
+        <div className="col-lg-8 col-md-12 col-sm-12 col-12">
+          <h1 className="font-size-main-heading">Add License</h1>
+          <br />
+          <form>
+            <InputContainer
+              value={""}
+              name=""
+              type="text"
+              id="admin-add-license-status"
+              onChange={handleChange}
+            >
+              Active
+            </InputContainer>
+            <InputContainer
+              value={""}
+              name=""
+              type="text"
```

Comment:
Should be a checkbox or Yes/No dropdown.

Reply:
@GMishx Updated!